src/governance.h

```diff
@@ -376,6 +376,7 @@ class CGovernanceManager
     void InitOnLoad();
 
     void RequestGovernanceObjectVotes(CNode* pnode);
+    void RequestGovernanceObjectVotes(std::vector<CNode*> vNodesCopy);
```

Comment:
Argument type should be const std::vector<CNode*>& instead of pass by value (saves copying of vector).